com.unity.formats.alembic/Editor/Importer/AlembicImporterEditor.cs

```diff
@@ -48,8 +48,13 @@ public override void OnInspectorGUI()
                     new GUIContent("Import Meshes", ""));
                 EditorGUILayout.PropertyField(serializedObject.FindProperty(pathSettings + "importPoints"),
                     new GUIContent("Import Points", ""));
-                EditorGUILayout.PropertyField(serializedObject.FindProperty(pathSettings + "importCurves"),
-                    new GUIContent("Import Curves", ""));
+                var importCurvesProp = serializedObject.FindProperty(pathSettings + "importCurves");
+                EditorGUILayout.PropertyField(importCurvesProp, new GUIContent("Import Curves", ""));
+                if (importCurvesProp.boolValue == true)
+                {
+                    EditorGUILayout.PropertyField(serializedObject.FindProperty(pathSettings + "createCurveRenderers"),
+                        new GUIContent("Add Curve Renderers", ""));
```

Comment:
Suggestion for tooltip: `Automatically add "Alembic Curves Renderer" components on curve objects. This component enables basic rendering of curves for preview purposes inside the Game and Scene views.`

I think we need to mention something like "preview" or "visualize" to make sure that users don't assume a full rendering solution (which I think some will).

Reply:
@sebastienduverne how does this sound?
*Automatically add "AlembicCurvesRenderer" components on curve objects. This component enables basic preview of curves.*

Reply:
I would just change the 2nd sentence:
`This allows you to preview curves in low resolution.`

Reply:
The preview isn't actually lower resolution. The curves will get rendered at the resolution of the scene/game view like all other objects in the scene.

Without this component though, the curves are never visible in the scene/game view. The main purpose of the component is to make the curves visible in the scene/game view but it shouldn't be used for "final quality render".

What makes this a "preview" is the limited quality/options for rendering. For example, a key feature of curves is the ability to modulate its width ([example](https://docs.arnoldrenderer.com/display/A5AFMUG/Curves#Curves-CurveWidth#Curves-CurveWidth)) but this is not possible with `AlembicCurvesRenderer` alone.

If a user wanted to control the width, they would have to code their own rendering solution (converting the curve data into visual information on screen). The demo team is also currently working on a bigger workflow (which includes Alembic as an importer) with more advanced rendering features. This is likely what users will be using in the future to get "good" curve renders inside Unity.

Reply:
Thanks for the clarification. Take 2 (2nd sentence only):
`This allows you to get a basic preview of the Alembic curves in the Scene.`

As for the curve features not available, this is typically nice details to have in documentation, I'll keep it in mind.